pkg/controllers/controllers.go

```diff
@@ -103,10 +103,11 @@ func Register(ctx context.Context, systemNamespace, controllerName string, cfg c
 	if len(systemNamespace) == 0 {
 		klog.Info("Starting helm controller with no namespace")
 	} else {
-		klog.Infof("Starting helm controller in namespace %s", systemNamespace)
+		klog.Infof("Starting %s in namespace %s", controllerName, systemNamespace)
```

Comment:
Maybe we should clarify these... `for namespace %s` and `for all namespaces with lock in %s` or something like that?

Reply:
Oh I like that idea a lot. Nice.

Reply:
Updated them now - until we add the feature to control lock namespace that's just a string. Once we add it we can swap that string for the variable `lockNamespace` (or w/e is implemented).